frontend/src/components/HacknightsParticipants.vue

```diff
@@ -59,7 +62,7 @@
       </v-toolbar>
       <v-divider v-if="getHacknight.participants"></v-divider>
       <v-list>
-        <template v-for="(item, i) in getHacknight.participants">
+        <template v-for="(item, i) in orderedParticipants()">
```

Comment:
Not a big deal in this case, but it would probably be more accurate to put `orderedParticipants` in `computed` instead of `methods`.

> Instead of a computed property, we can define the same function as a method. For the end result, the two approaches are indeed exactly the same. However, the difference is that computed properties are cached based on their reactive dependencies. A computed property will only re-evaluate when some of its reactive dependencies have changed.

From https://vuejs.org/v2/guide/computed.html

Reply:
I have tried that and it didn't work (offcourse I could mess somethnig up). Since we are recieving participants using getter (getHacknight) within computed I have had no access to participants from another method computed.

Reply:
I think it should be possible to use one computed inside another. Would `this` help? https://stackoverflow.com/questions/41173998/is-it-possible-to-use-the-computed-properties-to-compute-another-properties-in-v

Reply:
I would not - `this` is already there.  With that approach we are attempting to order participants which we have not got yet.

Reply:
I've checked out your branch, moved `orderedParticipants` to `computed` and it works fine if you wrap function body into `if(this.getHacknight) {...}`.